### PR TITLE
Batch evaluate ignore disabled flags

### PR DESCRIPTION
**Pull request by vic3lord**
When using BatchEvaluate the entire call is canceled with an error once it encounters the first disabled flag.
In order to be able to disable a flag without failing the entire request I added a new ErrDisabled and checked when doing BatchEvaluate.

This way you can continue batching flags with the possibility to disable flags.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -21,8 +21,8 @@ type Option func(s *Server)
 // Server serves the Flipt backend
 type Server struct {
 	logger logrus.FieldLogger
-
-	store storage.Store
+	store  storage.Store
+	flipt.UnimplementedFliptServer
```
Comment (markphelps) [inline server/server.go:25]: do you know the reasoning for why this was added?

**Review by markphelps — COMMENTED** [review]
makes sense @vic3lord , would you mind adding a test to validate the new behavior? also one comment/question

File: server/server.go (hunk 1 above)
Comment (vic3lord) [reply server/server.go:25]: Yeah, for some reason the code base was upgraded (thought it happened between my last PR) to use the new go protc plugin which in order to retain BC with the old package asks you to embed the unimplemented server.

**Comment by markphelps** [comment]
Thanks @vic3lord! I'll create a new release this weekend. I'm wondering if we should just not return errors when trying to evaluate a disabled flag in general (even for a single/non-batch evaluate call)?

**Comment by vic3lord** [comment]
I agree, we can discuss this further but I wanted to be conservative since you are maintainer, I didn't want to assume design decisions on my own.

We use flags internally as kill switches for new features that are not UI A/B testing.

Might make sense to not return an error in this case.

Thanks again for merging

**Pull request merged**